udata/features/territories/views.py

```diff
@@ -109,19 +109,39 @@ def render_territory(territory):
     if not current_app.config.get('ACTIVATE_TERRITORIES'):
         return abort(404)
 
-    DATASETS = TERRITORY_DATASETS[territory.level_name]
+    # Check that this territory is still valid (redirect old French
+    # regions for instance).
```

Comment:
Do we force redirect to new regions or do we advertise on the page that this territory is legacy with a link to the new one ?